src/sections/info.tsx

```diff
@@ -1,43 +1,96 @@
-      name: elem.ip,
-      value: index,
-      ip: elem.ip,
-      apiKey: elem.apiKey,
-    };
-  });
-  const connecting = useAppSelector(
-    (store: Store) => store.auth.status.connecting
-  );
-  const connected = useAppSelector(
-    (store: Store) => store.auth.status.connected
-  );
+  const { apiKey, ip } = useAppSelector(selector => selector.auth.loginData)
+  const balances = useAppSelector(selector => selector.sdk.balances)
+  const addresses = useAppSelector(selector => selector.sdk.addresses)
+  const channels = useAppSelector(selector => selector.sdk.channels)
+  const version = useAppSelector(selector => selector.sdk.version)
+  const info = useAppSelector(selector => selector.sdk.info)
+  const peers = useAppSelector(selector => selector.sdk.peers)
+
+  useEffect(() => {
+    if (apiKey && ip) {
+      dispatch(actionsAsync.getBalancesThunk({ apiEndpoint: ip, apiToken: apiKey }))
+      dispatch(actionsAsync.getChannelsThunk({ apiEndpoint: ip, apiToken: apiKey }))
+      dispatch(actionsAsync.getAddressesThunk({ apiEndpoint: ip, apiToken: apiKey }))
+      dispatch(actionsAsync.getVersionThunk({ apiEndpoint: ip, apiToken: apiKey }))
+      dispatch(actionsAsync.getInfoThunk({ apiEndpoint: ip, apiToken: apiKey }))
+      dispatch(actionsAsync.getPeersThunk({ apiEndpoint: ip, apiToken: apiKey }))
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+
+  // check if user is logged in
+  if (!apiKey || !ip) {
+    return <Section
+      className="Section--selectNode"
+      id="Section--selectNode"
+      yellow
+    >Login to node</Section>
+  }
+
 
   return (
     <Section
       className="Section--selectNode"
       id="Section--selectNode"
       yellow
-    ></Section>
+    >
+      <h2>Balances</h2>
+      <div id='balances'>
+        <div>
+          Native: {balances?.native}
+        </div>
+        <div>
+          Hopr: {balances?.hopr}
+        </div>
+      </div>
+      <h2>Info</h2>
+      <div id="info">
+        <div id='version'>
+          Version: {version}
+        </div>
+        {!!info && Object.entries(info).map(([key, value]) => {
+          if (Array.isArray(value)) {
+            return <div>
+              <h3>{key}</h3>
+              <ul>
+                {value.map(val => <li>{val}</li>)}
+              </ul>
+            </div>
+          }
+          return <div>
+            {key}: {value}
+          </div>
+        })}
+      </div>
+      <h2>Channels</h2>
+      <div id='channels'>
+        <div>
+          Incoming: {channels?.incoming.filter(channel => channel.status === 'Open').length}
+        </div>
+        <div>
+          Outgoing: {channels?.outgoing.filter(channel => channel.status === 'Open').length}
+        </div>
+      </div>
+      <h2>Peers</h2>
+      <div id='peers'>
+        <div>
+          Announced: {peers?.announced.length}
+        </div>
+        <div>
+          Connected: {peers?.connected.length}
+        </div>
+      </div>
```

Comment:
Lets add number of assigned aliases here too and also the whole node/sdk side in a <pre/> at least for the developement time. The latter part is also in the https://github.com/hoprnet/hopr-admin/pull/14 PR, so youshould get in on a rebase after 14 is merged